src/main/java/com/example/backendspringboottechiteasycontrollerles10/controllers/ExceptionController.java

```diff
@@ -0,0 +1,26 @@
+package com.example.backendspringboottechiteasycontrollerles10.controllers;
+
+import com.example.backendspringboottechiteasycontrollerles10.exceptions.RecordNotFoundException;
+import com.example.backendspringboottechiteasycontrollerles10.exceptions.TelevisionNameTooLongException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ExceptionController {
+    @ExceptionHandler
+    public ResponseEntity<String> exception (RecordNotFoundException exception) {
+        return new ResponseEntity<>(exception.getMessage(), HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler
+    public ResponseEntity<String> exception (IndexOutOfBoundsException exception) {
+        return new ResponseEntity<>(exception.getMessage(), HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler
+    public ResponseEntity<String> exception (TelevisionNameTooLongException exception) {
+        return new ResponseEntity<>(exception.getMessage(),HttpStatus.REQUEST_HEADER_FIELDS_TOO_LARGE);
```

Comment:
Dit is een rare HttpStatus keuze en niet juist. Je kunt hier misschien kiezen voor een BAD_REQUEST, dat is de algemene "client side error" httpstatus